pom.xml

```diff
@@ -32,6 +32,11 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.17</version>
         </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
```

Comment:
can we configure .properties file for logging?

Reply:
Ok, but in two hours ish